plugin/evm/limitorders/contract_events_processor_test.go

```diff
@@ -340,10 +340,10 @@ func TestHandleOrderBookEvent(t *testing.T) {
 		})
 		t.Run("When data in log unpack succeeds", func(t *testing.T) {
 			// order := getOrder(ammIndex, traderAddress, longOrder.BaseAssetQuantity, price, salt)
-			ordersMatchedEventData, _ := event.Inputs.NonIndexed().Pack(signature, fillAmount, big.NewInt(0).Mul(fillAmount, price), relayer, timestamp)
+			ordersMatchedEventData, _ := event.Inputs.NonIndexed().Pack(signature, fillAmount, price, big.NewInt(0).Mul(fillAmount, price), relayer, timestamp)
```

Comment:
`OrdersMatched` doesn't have a signature field, how is this working?

Reply:
The event is LiquidationOrdersMatched. I thought I fixed the var name when you highlighted this last time. My bad.